engine.go

```diff
@@ -148,6 +148,22 @@ func WithErrorHandler(errorHandler func(err error) error) func(*Engine) {
 	}
 }
 
+func WithTags(tags ...openapi3.Tag) func(*Engine) {
+	return func(e *Engine) {
+		for _, tag := range tags {
+			if tag.Name == "" {
+				panic("Tag name cannot be empty")
```

Comment:
Log an error or a warning, and continue

A panic here is dangerous

Reply:
Why? It's at startup

Reply:
+ it's a draft PR Christophe pleaaase 😂 So impatient 😊

Reply:
Panic is here. We could possibly move all of our panics to a fatal log. But yeah exiting here is best

Reply:
> +it's a draft PR Christophe pleaaase 😂 So impatient 😊

![395770858-718c5a79-c97e-4a60-bb00-c543221e328f.jpg](https://github.com/user-attachments/assets/384c7da9-4b5b-4ee4-98e1-1072961d184f)